docker/docker-compose.yml

```diff
@@ -43,11 +43,14 @@ services:
       BLOCK_INTERVAL: "50"
       EXPORT_TIMEOUT_MLS: 300000
       EXACT_CONTRACT_MODE: "1"
+      # Should be substituted from run.sh script with different values.
+      BLOCKCHAIN: ${BLOCKCHAIN}
```

Comment:
Wouldn't that make the file unusable w/o `run.sh`?

Reply:
Yes, do we have another use case of the file? What other way we have for providing dynamic variable?

Reply:
You can set a default value if the env var is not present:
```yaml
BLOCKCHAIN: ${BLOCKCHAIN:-ethereum}
```
https://docs.docker.com/compose/compose-file/compose-file-v3/#variable-substitution

Reply:
Default value provided in docker-compose.